python/cudf_polars/cudf_polars/dsl/expressions/unary.py

```diff
@@ -263,6 +265,22 @@ def do_evaluate(
                     plc.Column.from_scalar(arg, 1), column.obj.type()
                 ).to_scalar()
             return Column(plc.replace.replace_nulls(column.obj, arg), dtype=self.dtype)
+        elif self.name == "as_struct":
+            children = [
+                child.evaluate(df, context=context).obj for child in self.children
+            ]
+            return Column(
+                plc.Column(
+                    data_type=self.dtype.plc,
+                    size=children[0].size(),
```

Comment:
Do we know that `children` is nonempty here, so it's safe to do this `getitem`?

Reply:
I believe so, yes.

I think this call would be equivalent to having empty `children` which raises earlier

```python
In [1]: import polars as pl

In [2]: pl.LazyFrame({}).select(pl.struct(pl.all())).collect(engine="gpu")
...
InvalidOperationError: expected at least 1 input in null.as_struct()

Resolved plan until failure:

        ---> FAILED HERE RESOLVING 'select' <---
DF []; PROJECT */0 COLUMNS
```